src/main/app/src/components/Etusivu.tsx

```diff
@@ -68,8 +68,13 @@ export const Etusivu = () => {
   });
 
   const uutislinkit = uutiset?.['etusivun-uutiset']?.linkit ?? [];
+  const [showMore, setShowMore] = useState(true);
 
-  const [showMore, setShowMore] = useState(size(uutislinkit) <= 3);
+  useEffect(() => {
+    if (!isLoading) {
+      setShowMore(size(uutislinkit) > 3);
+    }
+  }, [isLoading]);
```

Comment:
Jos loppuun asti mietitään, tätä logiikkaahan ei kai enää tarvittaisi, kun joka ympäristössä uutislinkkejä on jo yli 3, eikä vanhoja kai ole tarkoitus tuhota. Pidetään nyt kuitenkin näin siltä varalta, että tarvetta on (?)